README.md

```diff
@@ -90,11 +90,12 @@ and the JSONata expression `Phone.{type: number}`, an object constructor, the ac
 ```Split Property``` - use this field to choose a separator.
 
 ### Re-assemble Messages 
-**(Beta)**
 
-Inverse of the split action: Given a stream of incoming messages that which have 
-been split apart by a split action (or similar), produce one message once all 
-message parts have arrived.
+Inverse of the split action: Given a stream of incoming messages a sum message is generated.
+Has 3 different behaviour variants:
+* Only specify group size and no delay timer. A message is emitted once the group size is reached for the given group. Should less message then the given group size arrive, then the group is silently discarded.
```

Comment:
> Should less message then the given group size arrive, then the group is silently discarded.

It is probably worth clarifying that if a message initially encounters an error and that error is resolved, then it could potentially re-join the batch.